scss/_patterns_accordion.scss

```diff
@@ -1,5 +1,4 @@
 @import 'settings';
```

Comment:
@anthonydillon Why are settings imported in every single pattern separately, instead of being part of the base?

We require base to be imported (and `@include vf-base`) before any pattern code, can't settings be part of that?
This `@import settings` gives a fake impression that pattern needs only settings, while they quite often use other mixins/patterns from base.
I guess we reached the point when we know we can't easily make patterns built completely independently from rest of Vanilla, so maybe we can make `@import settings` part of this already required base?

Reply:
This was from a point when we wanted all patterns to build without base.